src/main/java/com/google/cloud/discotoproto3converter/proto3/Proto3Writer.java

```diff
@@ -107,13 +107,22 @@ private void printServices(Collection<GrpcService> services, PrintWriter writer)
         writer.println(formatDescription("  ", method.getDescription()));
         writer.println("  " + method + " {");
 
-        StringBuilder optionsSb = new StringBuilder();
         for (Option option : method.getOptions()) {
-          optionsSb.append("    option (").append(option).append(") = {\n");
-          for (Map.Entry<String, String> prop : option.getProperties().entrySet()) {
-            optionsSb.append("      " + prop.getKey() + ": " + '"' + prop.getValue() + "\"\n");
-          }
+          StringBuilder optionsSb = new StringBuilder();
+          optionsSb.append("    option (").append(option).append(") = ");
+          
+          if (option.toString().equals("google.api.http")) {
```

Comment:
Now we're using strings for both `Option` construction in the other file and comparison here. I think it makes sense to use an enum of allowed `Option`s, and we refer to the enum value when creating and comparing the `Option` type. This allows the compiler to help us catch typos, and is more elegant. If you can do that now, great, if not, I suggest a TODO and we can do it in a subsequent PR.